pkg/loader/openapi.go

```diff
@@ -138,7 +150,8 @@ func getOpenAPITools(t *openapi3.T) ([]types.Tool, error) {
 				}
 
 				if bodyMIME == "" {
-					return nil, fmt.Errorf("no supported MIME types found for request body in operation %s", operation.OperationID)
+					// No supported MIME types found, so just skip this operation and move on.
+					continue operations
```

Comment:
We now skip tools with no supported MIME types in the body instead of just returning an error for the whole document.